test/rop.jl

```diff
@@ -44,161 +44,160 @@
             @test isapprox(gen_status(result,"2","3"), 1; atol=1e-6)
             @test isapprox(gen_status(result,"2","4"), 1; atol=1e-6)
-            @test isapprox(gen_status(result,"3","1"), 1; atol=1e-6)
-            @test isapprox(gen_status(result,"3","2"), 1; atol=1e-6)
-            @test isapprox(gen_status(result,"3","3"), 1; atol=1e-6)
-            @test isapprox(gen_status(result,"3","4"), 1; atol=1e-6)
 
+            @test isapprox(branch_status(result,"0","1"), 0; atol=1e-6)
+            @test isapprox(branch_status(result,"0","2"), 0; atol=1e-6)
+            @test isapprox(branch_status(result,"0","4"), 1; atol=1e-6)
             @test isapprox(branch_status(result,"1","1"), 0; atol=1e-6)
             @test isapprox(branch_status(result,"1","2"), 0; atol=1e-6)
             @test isapprox(branch_status(result,"1","4"), 1; atol=1e-6)
-            @test isapprox(branch_status(result,"2","1"), 0; atol=1e-6)
-            @test isapprox(branch_status(result,"2","2"), 0; atol=1e-6)
-            @test isapprox(branch_status(result,"2","4"), 1; atol=1e-6)
             # @test isapprox(branch_status(result,"3","1"), 1; atol=1e-6)
-            @test isapprox(branch_status(result,"3","2"), 1; atol=1e-6)
-            @test isapprox(branch_status(result,"3","4"), 1; atol=1e-6)
+            # @test isapprox(branch_status(result,"2","2"), 0; atol=1e-6)
+            @test isapprox(branch_status(result,"2","4"), 1; atol=1e-6)
 
-            @test isapprox(load_power(result, "1",["1","2","3"]), 4; atol=1e-2)
-            @test isapprox(load_power(result, "2",["1","2","3"]), 7; atol=1e-2)
-            @test isapprox(load_power(result, "3",["1","2","3"]), 10; atol=1e-2)
+            @test isapprox(load_power(result, "0",["1","2","3"]), 4; atol=1e-2)
+            @test isapprox(load_power(result, "1",["1","2","3"]), 7; atol=1e-2)
+            @test isapprox(load_power(result, "2",["1","2","3"]), 10; atol=1e-2)
 
-            @test isapprox(gen_power(result, "1",["1","2","3","4","5"]), 4; atol=1e-2)
-            @test isapprox(gen_power(result, "2",["1","2","3","4","5"]), 7; atol=1e-2)
-            @test isapprox(gen_power(result, "3",["1","2","3","4","5"]), 10; atol=1e-2)
+            @test isapprox(gen_power(result, "0",["1","2","3","4","5"]), 4; atol=1e-2)
+            @test isapprox(gen_power(result, "1",["1","2","3","4","5"]), 7; atol=1e-2)
+            @test isapprox(gen_power(result, "2",["1","2","3","4","5"]), 10; atol=1e-2)
 
         end
 
-        mn_data = build_mn_data(case5_restoration_strg, replicates=3)
+        mn_data = build_mn_data(case5_restoration_strg, replicates=2)
         @testset "5-bus strg case" begin
             result = PowerModelsRestoration.run_rop_uc(mn_data, PowerModels.DCPPowerModel, cbc_solver)
 
             @test result["termination_status"] == OPTIMAL
-            @test isapprox(result["objective"], 701.00; atol = 1e-2)
+            @test isapprox(result["objective"], 1934.0; atol = 1e-2)
```

Comment:
Any idea why the objective changes so much?

Reply:
Here, the solution is ~ 3x and expected